src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.first_bar {
+  position: absolute;
+  width: 900px;
+  height: 70px;
+  left: 48px;
+  top: 71px;
+}
+
+.icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  display: block;
+}
+
+body {
+  font-weight: 300;
+  color: rgba(61, 78, 97, 1);
+}
```

Comment:
fonts better to move to html tag

and fo other styles use classes

Reply:
I moved it, but I'm not sure about the color. Does it also need to be moved?